sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -1143,7 +1142,7 @@ export class KeyClient {
         expiresOn: attributes.expires,
         createdOn: attributes.created,
         updatedOn: attributes.updated,
-        vaultUrl: parsedId.vaultUrl,
+        vaultUrl: parsedId.vaultUri,
```

Comment:
If "url" is more idiomatic than "uri" (for everyone but .NET, I thought it was) then use that. Basically, whatever the property/parameter on clients is.

Reply:
Addressed 🌞